src/client/client.go

```diff
@@ -47,6 +47,8 @@ type APIClient struct {
 var (
 	//BlockSize is used internally by PFS blockserver
 	BlockSize = 8 * 1024 * 1024 // 8 Megabytes
+	// MaxBlockSize specifies the maximum block size for any data type
+	MaxBlockSize = 100 * 1024 * 1024 // 100 MB
 	// MaxMsgSize is used to define the GRPC frame size, which we need to be greater than a block
 	MaxMsgSize = 3 * BlockSize
```

Comment:
I don't understand. Even in the system without that buffering change you could send a block as multiple small messages or you could send multiple blocks as a single large message. The 2 things seem completely decoupled to me. MaxMsgSize should affect how we break things up in streaming calls but it doesn't seem like it should be related to the size we use for blocks.